definitions/region/european-regions.yaml

```
@@ -72,3 +72,33 @@
       definition: EU27 excluding CY + MT
 
   # add more definitions of `EU*` above this line
+
+  # region definition from the INNOPATHS project
+- INNOPATHS regions:
+  - Germany (IP):
+      innopaths: DEU
+      countries: Germany
+  - Eastern Europe (IP):
+      innopaths: ECE
+      countries: Czech Republic, Estonia, Latvia, Lithuania, Poland, Slovakia
+  - Scandinavia (IP):
+      innopaths: ENC
+      countries: Denmark, Finland, Sweden
+  - South-East Europe (IP):
+      innopaths: ECS
+      countries: Bulgaria, Croatia, Hungary, Romania, Slovenia
+  - Southern Europe (IP):
+      innopaths: ESC
+      countries: Cyprus, Greece, Italy, Malta
+  - Iberian Peninsula (IP):
+      innopaths: ESW
+      countries: Portugal, Spain
+  - Central Europe (IP):
+      innopaths: EWN
+      countries: Austria, Belgium, Luxembourg, Netherlands
+  - France (IP):
+      innopaths: FRA
+      countries: France
+  - United Kingdom & Ireland (IP):
+      innopaths: UKI
+      countries: UK, Ireland
```

For the sake of completeness I would add the following two extra regions.

```suggestion
      countries: UK, Ireland

- European (non-EU) north countries (IP):
      innopaths: NEN
      countries: Iceland, Liechtenstein, Norway, Switzerland

- European (non-EU) south countries (IP):
      innopaths: NES
      countries: Albania, Andorra, Bosnia and Herzegovina, North Macedonia, Monaco, Montenegro, Serbia, Turkey
```

Maybe you can come up with a better naming convention than this. 
The NEN region can be specially important as it includes non-eu countries that are part of the ETS (Iceland, Liechtenstein, Norway) or have an ETS system in place (Switzerland).

---

Thanks @Renato-Rodrigues!